File: 250718/남녀노소 구분짓기/sex-and-age.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int gender = sc.nextInt();
        int age = sc.nextInt();

        // 조건문
        if (gender == 0) { // 남자
             if (age >= 19) {
                System.out.println("MAN");
             } else {
                System.out.println("BOY");
             }
        } else { // 여자
            if (age >= 19) {
                System.out.println("WOMAN");
            } else {
                System.out.println("GIRL");
            }
        }
    }
}

File: 250718/남녀노소 구분짓기/sex-and-age.py
# 입력
gender = int(input())
age = int(input())

# 조건문
if gender == 0 : # 남자
    if age >= 19 :
        print("MAN")
    else :
        print("BOY")
else : # 여자
    if age >= 19 :
        print("WOMAN")
    else :
        print("GIRL")
